testsuite/tests/apicast/policy/maintenance_mode/test_maintenance_mode_policy_host.py

```diff
@@ -20,16 +21,22 @@
     pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-6552")]
 
 
+@pytest.fixture
+def echo_api_base_url(private_base_url):
+    """Return URL of echo_api"""
+    return private_base_url("echo_api")
```

Comment:
Since you are already changing this, wouldn't it make sense to changing to the actual backend and not an alias?

Reply:
No, this should be working also in evironments where echo_api is echo-api (maybe saas)